core/app/c/(public)/[communitySlug]/public/forms/[formSlug]/fill/ExternalFormWrapper.tsx

```diff
@@ -30,6 +31,7 @@ export const ExternalFormWrapper = ({
 	const router = useRouter();
 	const pathname = usePathname();
 	const params = useSearchParams();
+	const [pubId] = useState(props.pub.id);
```

Comment:
sneaky bug where if you used the member select on a new form the server would make a new pubid but we were not caching it here (but are caching it in `PubEditorClient` so the pubids become out of sync and we make a pub with a different id than the one we navigate to